packages/web3/src/TestGatingNFT.ts

```diff
@@ -233,7 +110,7 @@ export async function publicMint(nftName: string, toAddress: Address): Promise<n
     throw Error('No mint event found')
 }
 
-export async function burn(nftName: string, tokenId: number): Promise<void> {
+async function burn(nftName: string, tokenId: number): Promise<void> {
```

Comment:
just a comment, i'm pretty sure all usages of createTestClient should use a throwaway account, but we can do that in a follow up pr.

Reply:
* especially any usage of createTestClient that writes to a contract. Now I think we should fix this here.

Reply:
How come?

Reply:
(refactored)